lib/util/http/client/type.go

```diff
@@ -18,6 +18,7 @@ import (
 
```

Comment:
Should I also propagate the changes from the `ParseResponse ` method in `internal/http/client.go` to the same method here? I believe it got left in this file for backwards compatibility, in case someone depends on it in some plugin.

Reply:
I think we can safely leave it, I'm not sure how many people are actually using it in the wild but I don't think we need to go as far as updating it.